vnext/Microsoft.ReactNative/pch.h

```diff
@@ -3,9 +3,46 @@
 
 #pragma once
 
+//#include <SDKDDKVer.h>
+//
+#ifndef WIN32_LEAN_AND_MEAN
+#define WIN32_LEAN_AND_MEAN
+#endif
+
+#ifndef NOGDI
+#define NOGDI
+#endif
+//
+//#include <windows.h>
+//
+//#include <Unknwn.h>
+
 #include <hstring.h>
+#include <oaidl.h>
+#include <werapi.h>
 #include <winrt/Windows.ApplicationModel.Activation.h>
 #include <winrt/Windows.Foundation.Collections.h>
 #include <winrt/Windows.Foundation.h>
+#include <winrt/Windows.System.h>
+#include <winrt/Windows.UI.Composition.h>
+#include <winrt/Windows.UI.Core.h>
+#include <winrt/Windows.UI.Xaml.Controls.Primitives.h>
+#include <winrt/Windows.UI.Xaml.Controls.h>
+#include <winrt/Windows.UI.Xaml.Media.h>
+#include <winrt/Windows.Web.Http.Headers.h>
 #include "HResult.h"
```

Comment:
>"H [](start = 9, length = 2)

(nit) should this also be in "<>" brackets?

Reply:
No, it is a local file.

---
In reply to: [360531923](https://github.com/microsoft/react-native-windows/pull/3809#discussion_r360531923) [](ancestors = 360531923)